### PR TITLE
feat(index.js): adjust display date

### PR DESCRIPTION
**Pull request by yc97463**

### CODE REVIEW

**Comment by pan93412** [comment]
這個語法或許可以讓實作更漂亮一點（雖然好像變長了 😂）

    today.toLocaleDateString('zh-tw', { day: 'numeric', month: 'numeric' })

---

image

**Review by pan93412 — COMMENTED** [review]
這個語法或許可以讓實作更漂亮一點（雖然好像變長了 😂）

    today.toLocaleDateString('zh-tw', { day: 'numeric', month: 'numeric' })

image

**Review by liaojason2 — CHANGES_REQUESTED** [review]

**Comment by liaojason2** [comment]
I don't think this a neccessary change, but @justinlin033 said whatever, so I think I can merge this PR after you fix your code. 

Screen Shot 2022-09-27 at 1 48 07 AM

File: index.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ bot.onText(/^\/poll(?:@.*?)?( .*|$)/, function(msg, match) {
     else if (!time.match(timeRegex)) {
       bot.sendMessage(chatId, 'Justin, 這不是時間');
     } else {
-      message = `今天 ${time} 要吃拉麵嗎`;
+      message = `${today.getMonth()+1}/${today.getDate()} ${time} 要吃拉麵嗎`;
```
[the remark this answers is not in the document]
Comment (yc97463) [reply index.js]: Oops, I forgot to define `today`.

File: index.js [hunk 2]
```diff
@@ -72,7 +72,8 @@ bot.onText(/^\/poll(?:@.*?)?( .*|$)/, function(msg, match) {
     else if (!time.match(timeRegex)) {
       bot.sendMessage(chatId, 'Justin, 這不是時間');
     } else {
-      message = `今天 ${time} 要吃拉麵嗎`;
+      const today = new Date();
+      message = `${today.toLocaleDateString('zh-tw', { day: 'numeric', month: 'numeric' })} ${time} 要吃拉麵嗎`;
```
Comment (liaojason2) [inline index.js:76]: Line length too long
https://eslint.org/docs/latest/rules/max-len

Comment (liaojason2) [inline index.js:76]: Line length too long
https://eslint.org/docs/latest/rules/max-len

**Review by liaojason2 — CHANGES_REQUESTED** [review]

**Review by liaojason2 — APPROVED** [review]

**Pull request merged**